core/new-gui/src/app/workspace/service/execute-workflow/execute-workflow.service.ts

```diff
@@ -80,7 +80,7 @@ export class ExecuteWorkflowService {
             break;
           } default: {
             // workflow status related event
-            if (event.type !== 'WorkflowStatusUpdateEvent') {
+            if (event.type !== 'WebWorkflowStatusUpdateEvent') {
```

Comment:
can those type name strings be declared as variables?

Reply:
In typescript, these strings are typed as enumerations (unlike java). We use string value as enum a lot in the frontend because plain JS doesn't have enum.